### ISSUE TITLE
goto not working

### ISSUE THREAD

**Issue by bsherb53** [db-journey/cassandra-driver#6]
if I have version 1, 2, 3, 4, 5 (all created in that order), and I am currently at 5. If I use `migrate goto 3`, it applies the down for all versions.

**Comment by gravis** [comment]
You're right, `goto` is completely broken. It was based on mattes/migrate, which is (was?) using sequential migration number. Now that we're using timestamps instead, it won't work as it, the code must be rewritten. Sorry for that, I'll try to find the time to fix this, or you can provide a PR to fix the behaviour.
Thanks
